test/game_stats_test.rb

```diff
@@ -7,57 +7,51 @@
 class GameStatsTest < MiniTest::Test
 
   def setup
-    locations = { games: './data/games.csv', teams: './data/teams.csv', game_teams: './data/game_teams.csv' }
+    locations = { games: './data/dummy_games.csv', teams: './data/teams.csv', game_teams: './data/dummy_game_teams.csv' }
     @stat_tracker = StatTracker.from_csv(locations)
   end
 
   def test_highest_total_score
-    assert_equal 11, @stat_tracker.highest_total_score
+    assert_equal 5, @stat_tracker.highest_total_score
   end
 
   def test_lowest_total_score
-    assert_equal 0, @stat_tracker.lowest_total_score
+    assert_equal 3, @stat_tracker.lowest_total_score
   end
 
   def test_biggest_blowout
-    assert_equal 8, @stat_tracker.biggest_blowout
+    assert_equal 3, @stat_tracker.biggest_blowout
   end
 
   def test_percentage_home_wins
-    assert_equal 0.44, @stat_tracker.percentage_home_wins
+    assert_equal 0.5, @stat_tracker.percentage_home_wins
```

Comment:
Should this return a float to the 100th?

Reply:
a float, the data-type was there all I did was change the number.